from context.shared.context_error import ContextError

class ContextTextError( ContextError ):

    pass

#-- END class ContextTextError --#
